README.md

```diff
@@ -0,0 +1,81 @@
+# Python Example
+
+This is a template project to jump-start the creation of additional project repositories, managed by Blur Studio, on GitHub. Below is a walk-through of various characteristics of the project.
+
+## Package Configuration
+
+### `setup.py`
+
+- Enables the `setuptools` extension `setuptools_scm`. This extension replaces the need to statically define a package's version and instead derives the version from SCM metadata (such as the latest git tag).
+- Permit package installation in editable mode. (i.e. `pip install -e .`)
+
+_You should not need to modify the contents `setup.py`._
+
+> _Originally `setup.py` was the primary manor by which to define a package's metadata. It has mostly been replaced by `setup.cfg`._
+
+### `setup.cfg`
+
+- Defines static package metadata such as package description, author, and PyPI classifiers.
+- Lists package requirements (`install_requires`).
+- Lists auxiliary package requirements (`extras_require`). These can be installed by appending the extras group name (or "all" for all groups) in square-brackets upon install (ex: `pip install -e .[lint]`).
```

Comment:
I wonder if adding a example of specifying multiple would be more useful? `.[lint,format]`. It lets you know how to specify multiple without needing to figure out how to search for it. It should be easier to work out how to do a single specification from a multiple specification, but we could just have both examples.